### ISSUE TITLE
Issue with Torch Dependencies

### ISSUE THREAD

**Issue by nn-learner** [SeanNaren/CTCNetworkClient#88]
Hi Sean,

I don't know whether you are still working on maintaining this repo. However, I recently had to upgrade environments and it removed my torch libraries. I'm have an issue with the dependencies. It seems cunnx is no longer supported by the latest version of torch. I was wondering if you knew of a work around.

**Comment by SeanNaren** [comment]
I don't think `cunnx` is used in this repo, could you point me to where it is?

**Comment by nn-learner** [comment]
It says on the wiki for the required dependencies to compile Baidu's CTC.

**Comment by SeanNaren** [comment]
Thanks for that, it should not be necessary (but nnx is)! I've removed that from the install at the wiki page.

**Comment by nn-learner** [comment]
Interesting so when I try to transcribe the audio sample with one of my pre-trained weights, I get the following output:

`th Predict.lua -audioPath audio_samples/1089-134691-0009.flac 
mspsbsfsspsfsmsmspsysnsmsmspsmspsmsmpbgbysssmsysymsmskdspmbmbspkspdbsksmydysmsgpysklbddsbd	`

I have been using this pretrained weight before in my other environment and it transcribed perfectly fine. Any suggestions why this might be occuring?

Also, I have been moving these torch dependencies to a docker, which I can share with you if you want to add it to your repo documentation.

**Comment by SeanNaren** [comment]
Is the dictionary correct? Seems like something's up with the decoding

**Comment by nn-learner** [comment]
Yes, I tried both on a fresh git pull from your repo and my existing instance of the code I pulled.
